service/awsconfig/v4/error.go

```diff
@@ -1,4 +1,4 @@
-package v3
+package v4
```

Comment:
I fixed the script. I'll share in https://github.com/giantswarm/giantswarm/issues/2335